<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>theVirtualThings - VM Creation page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Virtual Machine - Creation</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="guiLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>431</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Does the VM needs to run a graphical interface ? (XFCE 4)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="guiCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes </string>
    </property>
   </widget>
   <widget class="QLabel" name="network1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>441</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Does the VM needs to be connected to a Virtual Network ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="isolatedLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>431</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Does the VM needs to be isolated in a private network ?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isolatedCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="networkLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="network2Label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Empty if you don't want to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QComboBox" name="imageComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select your image :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
